hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -259,7 +259,15 @@ public void doStateTransitionOperation(
             txnCtx.setTargetedContract(newContractId);
             sigImpactHistorian.markEntityChanged(newContractId.getContractNum());
         } else {
-            recordService.externalizeUnsuccessfulEvmCreate(result);
+            if (properties.enabledSidecars().contains(SidecarType.CONTRACT_BYTECODE)
+                    && op.getInitcodeSourceCase() != INITCODE) {
```

Comment:
Shouldn't we also track failing init bytes which are less than the Hedera maximum limit (not uploaded in File)?